### DISCUSSION TITLE
Is there a way to continue parallel deployment even if one host encounter an error?

### DISCUSSION THREAD

**Discussion by Tristan-MyAnaPro** [deployphp/deployer#3668] [Help needed]
Hi !

I use deployer V7 with 10 hosts and parallel deployment is perfect !

However, yesterday I had a problem where one of the servers was down, and I discovered that if a single command fails for one host, ALL the deployment is canceled and every hosts stops at the step failed.

Is there a way to tell deployer to call the `fail()` callback for this host but continue normally for the others healthy hosts?

Thx for your help! :)

**Comment by antonmedv** [comment]
I think right now there is no such feature. But  I can add it for sure.

Right now you can just exclude this host from deploy:

```sh
dep deploy 'alias!=failing-host'
```

**Reply by Tristan-MyAnaPro** [reply]
Okay thanks for the quick answer!

In my case I've used a similar selector to exclude the failing host and it's all good.

I think this could be a cool feature to have (with a flag like `--allow-failure` or `--continue-on-fail` idk) especially on CI/CD where we deploy many preview environment and are OK if some fails.

Should I mark this discussion as answered or do you like to check the feasibility of this feature beforehand?

**Reply by antonmedv** [reply]
Created a tracking issue https://github.com/deployphp/deployer/issues/3671
